src/views/searchview/search-view-list-header.js

```diff
@@ -6,7 +6,7 @@ import sharedStyles from '../data/data-view-shared.styles';
 @customElement('search-view-list-header')
 class SearchViewListHeader extends LitElement {
   @property({ type: String }) searchQuery;
-
+  @property({ type: String }) resultNumber;
   static get styles() {
```

Comment:
Can you keep an empty line above each function? That will makes things much more readable.